classes/rust.bbclass

```diff
@@ -1,7 +1,6 @@
 RUSTC = "rustc"
 
-# FIXME: --sysroot might be needed
-RUSTC_ARCHFLAGS += "--target=${TARGET_SYS} -C rpath"
+RUSTC_ARCHFLAGS += "--target=${TARGET_SYS} --sysroot=${STAGING_DIR_NATIVE}/${prefix_native} -C rpath"
```

Comment:
Without this, I was seeing failures along the lines of "failed to find target x86_64-linux.json".  The --bindir patch was previously causing the default sysroot to be correct, however I had difficulties with that patch, see below...